core/trino-main/src/test/java/io/trino/sql/gen/TestDynamicPageFilter.java

```diff
@@ -283,27 +283,27 @@ public void testDynamicFilterUpdates()
                 createLongSequenceBlock(100, 201),
                 createLongSequenceBlock(200, 301));
 
-        FilterEvaluator filterEvaluator = pageFilter.createDynamicPageFilterEvaluator(COMPILER).get();
+        FilterEvaluator filterEvaluator = pageFilter.createDynamicPageFilterEvaluator(COMPILER, dynamicFilter).get();
         verifySelectedPositions(filterPage(page, filterEvaluator), 101);
 
         dynamicFilter.update(TupleDomain.withColumnDomains(
                 ImmutableMap.of(columnB, multipleValues(BIGINT, ImmutableList.of(131L, 142L)))));
-        filterEvaluator = pageFilter.createDynamicPageFilterEvaluator(COMPILER).get();
+        filterEvaluator = pageFilter.createDynamicPageFilterEvaluator(COMPILER, dynamicFilter).get();
         verifySelectedPositions(filterPage(page, filterEvaluator), new int[] {31, 42});
 
         dynamicFilter.update(TupleDomain.all());
         dynamicFilter.update(TupleDomain.withColumnDomains(
                 ImmutableMap.of(columnC, singleValue(BIGINT, 231L))));
-        filterEvaluator = pageFilter.createDynamicPageFilterEvaluator(COMPILER).get();
+        filterEvaluator = pageFilter.createDynamicPageFilterEvaluator(COMPILER, dynamicFilter).get();
         verifySelectedPositions(filterPage(page, filterEvaluator), new int[] {31});
 
         dynamicFilter.update(TupleDomain.all());
-        Supplier<FilterEvaluator> filterEvaluatorSupplier = pageFilter.createDynamicPageFilterEvaluator(COMPILER);
+        Supplier<FilterEvaluator> filterEvaluatorSupplier = pageFilter.createDynamicPageFilterEvaluator(COMPILER, dynamicFilter);
         verifySelectedPositions(filterPage(page, filterEvaluatorSupplier.get()), new int[] {31});
 
         assertThat(dynamicFilter.isComplete()).isTrue();
         // After dynamic filter is complete, we should get back the same cached FilterEvaluator supplier
-        assertThat(pageFilter.createDynamicPageFilterEvaluator(COMPILER)).isEqualTo(filterEvaluatorSupplier);
+        assertThat(pageFilter.createDynamicPageFilterEvaluator(COMPILER, dynamicFilter)).isEqualTo(filterEvaluatorSupplier);
     }
 
```

Comment:
Please include the test from https://github.com/raunaqmorarka/presto/commit/08bfad13859e85babb4495e6c5ca380e7292ee05#diff-c8569f8c76ac854fbbb505be1a5a1a0177496697381e5e0779e447a0eb08a449R309